### PR TITLE
implement asset cropping and other updates

### PR DESCRIPTION
**Pull request by hburgund**
- implement asset active region in audiotrack playback mechanism
- turn off complex recording mechanism as it screws up audio session initialization
- update panning algorithm
- apply track-level filtering; filters already built, but not applied to filter chain yet

### CODE REVIEW

**Comment by hburgund** [comment]
This PR is now ready for your review @loafofpiecrust

File: RWFramework/RWFramework/Playlist/Playlist.swift [hunk 1]
```diff
@@ -77,7 +77,7 @@ class Playlist {
         audioMixer.distanceAttenuationParameters.rolloffFactor = 0.00001
         audioMixer.distanceAttenuationParameters.referenceDistance = 1
         audioMixer.distanceAttenuationParameters.maximumDistance = 200_000
-        audioMixer.renderingAlgorithm = .soundField
+        audioMixer.renderingAlgorithm = .HRTFHQ
```
Comment (loafofpiecrust) [inline RWFramework/RWFramework/Playlist/Playlist.swift:80]: This PR works fine for me except the `HRTFHQ` rendering doesn't seem to work in the simulator (😭 ) but if it works on device, then this is all set.

**Review by loafofpiecrust — APPROVED** [review]

File: RWFramework/RWFramework/Playlist/Playlist.swift [hunk 2]
```diff
@@ -236,7 +236,7 @@ extension Playlist {
             for it in data {
                 // TODO: Try to remove playlist dependency. Maybe pass into method?
                 it.playlist = self
-                it.player.renderingAlgorithm = .soundField
+                it.player.renderingAlgorithm = .HRTFHQ
```
Comment (loafofpiecrust) [inline RWFramework/RWFramework/Playlist/Playlist.swift]: We need to remove this line entirely. It's unnecessary and messes up for me with `.HRTFHQ`.
```suggestion
```

**Review by loafofpiecrust — CHANGES_REQUESTED** [review]

**Comment by hburgund** [comment]
@loafofpiecrust I have rebased in the change you requested after testing on my end, so this should be good to merge now, I think

**Review by loafofpiecrust — APPROVED** [review]

**Pull request merged**